todo_app/data/session_items.py

```diff
@@ -67,3 +68,20 @@ def save_item(item):
     session['items'] = updated_items
 
     return item
+
+def delete_item(id):
+    
+    existing_items = get_items()
+    item = get_item(id)
+    
+    existing_items.remove(item)
+
+    updated_items = []
+
+    for item in existing_items:
+        if(item['title'] != None):
+            updated_items.append(item)
+    
+    session['items'] = updated_items
```

Comment:
This is a little odd - removing items with blank titles as a side effect of removing the item by id?

Perhaps you did this because you had issues with getting the function to work. All you needed was to re-assign to `session['items']` to prompt Flask to update the user's cookie.
```suggestion
    session['items'] = existing_items
```